ChangeLog.md

```diff
@@ -1,10 +1,13 @@
 Change log
 ==========
 
-v0.6.0 (9Sep20)
+v0.6.0 (10Sep20)
 -----------------------
 
 - Numerical computation of the Hessian matrix 
+- Calculator for polydisperse systems 
```

Comment:
polydisperse systems is unclear, say something about the interaction potential. Is is generic pair potentials?